Gulpfile.js

```diff
@@ -19,6 +21,11 @@ const files = {
     js: './src',
     static: ['./www/**/*', './config.xml', './electron.js', '!./www/**/*.scss'],
     sass: './styles/*.scss',
+    androidApk: './platforms/android/build/outputs/apk',
+    unalignedApk: './platforms/android/build/outputs/apk/android-release-unaligned.apk',
+    alignedApk: './platforms/android/build/outputs/apk/android-release-aligned.apk',
+    zipAligned: '/Users/nuru/Library/Android/sdk/build-tools/23.0.3/zipalign', // Note the path to the zipalign on your pc
```

Comment:
Two minor comments:

* zipAligned implies an .apk that has been zip-aligned. This is the tool, so better to call it `zipAlign`, or just `zipalign` to match the actual file name (and update line 125 as well)
* Use the `ANDROID_SDK_ROOT` env var instead of hardcoding the full path (e.g. `require('gulp-util')`, then `gutil.env.ANDROID_SDK_ROOT`)